macro/src/lib.rs

```diff
@@ -79,9 +79,22 @@
 //!
 //! ```ignore
 //! #[subxt::subxt(crate = "crate::path::to::subxt")]
+//! pub mod polkadot {}
 //! ```
 //!
 //! By default the path `::subxt` is used.
+//!
+//! ### Expose documentation
+//!
+//! In order to expose the documentation from the runtime metadata on the generated
+//! code, users must specify the `expose_documentation` flag:
+//!
+//! ```ignore
+//! #[subxt::subxt(expose_documentation)]
```

Comment:
Nit: I feel like something like `generate_docs` would make more sense, since that's really what's happening here

Reply:
I agree that generate_docs is a better choice:)

Reply:
I tweaked this quickly via the in-browser editor :)